zipline/data/minute_bars.py

```diff
@@ -549,7 +549,9 @@ def last_date_in_output_for_sid(self, sid):
         with open(sizes_path, mode='r') as f:
             sizes = f.read()
         data = json.loads(sizes)
-        num_days = data['shape'][0] / self._minutes_per_day
+        # use integer division so that the result is an int
+        # for pandas index later https://goo.gl/VAW649
```

Comment:
Also you should squash your commits into 1, and change the commit message to say **BUG: <description>** (to be consistent with our dev guidelines https://github.com/quantopian/zipline/wiki/Zipline-development-guidelines)